### PR TITLE
Increase refresh interval default and introduce new 2m duration option

### PR DESCRIPTION
**Pull request by jshaughn**
Part-of https://github.com/kiali/kiali/issues/5269

### CODE REVIEW

**Review by Xunzhuo — COMMENTED** [review]
Why set 60s to default value not 30s?

File: crd-docs/crd/kiali.io_kialis.yaml [hunk 1]
```diff
@@ -1025,9 +1025,9 @@ spec:
                                 type: string
                                 enum: ["none","sent","received","total"]
                       metrics_per_refresh:
-                        description: "Duration of metrics to fetch on each refresh. Value must be one of: `1m`, `5m`, `10m`, `30m`, `1h`, `3h`, `6h`, `12h`, `1d`, `7d`, or `30d`"
+                        description: "Duration of metrics to fetch on each refresh. Value must be one of: `1m`, '2m', `5m`, `10m`, `30m`, `1h`, `3h`, `6h`, `12h`, `1d`, `7d`, or `30d`"
```
Comment (jmazzitelli) [inline crd-docs/crd/kiali.io_kialis.yaml:1028]: wrong ticks
```suggestion
                        description: "Duration of metrics to fetch on each refresh. Value must be one of: `1m`, `2m`, `5m`, `10m`, `30m`, `1h`, `3h`, `6h`, `12h`, `1d`, `7d`, or `30d`"
```

**Review by jmazzitelli — CHANGES_REQUESTED** [review]

**Comment by jshaughn** [comment]
> Why set 60s to default value not 30s?

@Xunzhuo Remember, this is not the scrape interval, this is just the default refresh interval for the Refresh dropdown menu.  15s is usually too frequent, except for maybe demos of a small mesh.  I chose 1m to reduce default refresh by 4x, something a little more meaningful.  Note that the default is often not even used, as the URL will override it. And, of course, it is easy to change via the dropdown.

**Comment by Xunzhuo** [comment]
> > Why set 60s to default value not 30s?
> 
> 
> 
> @Xunzhuo Remember, this is not the scrape interval, this is just the default refresh interval for the Refresh dropdown menu.  15s is usually too frequent, except for maybe demos of a small mesh.  I chose 1m to reduce default refresh by 4x, something a little more meaningful.  Note that the default is often not even used, as the URL will override it. And, of course, it is easy to change via the dropdown.
> 
> 

Yes, I realized just now 😂, this is more like the frequency from front end to call backend to refresh the graph.

**Review by jmazzitelli — APPROVED** [review]

**Pull request merged**